app-admin/etcd-wrapper/files/etcd-member.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=etcd (System Application Container)
+Documentation=https://github.com/coreos/etcd
+Conflicts=etcd.service
+Conflicts=etcd2.service
+
+[Service]
+Type=notify
+Restart=always
```

Comment:
not sure if this is a good choice. if user removed the local member, etcd will exit. but systemd will restart it. this wont cause etcd problem, but etcd will fall into restart loop.